### ISSUE TITLE
threaded discussions on procedure pages

### ISSUE THREAD

**Issue by mooreniemi** [mooreniemi/Transbucket_Rails#53]

**Comment by mooreniemi** [comment]
https://github.com/elight/acts_as_commentable_with_threading

**Comment by mooreniemi** [comment]
should open up threading on pins?

**Comment by mooreniemi** [comment]
![screenshot 2016-07-12 16 56 26](https://cloud.githubusercontent.com/assets/1695630/16783339/9af1509c-4851-11e6-8895-7f5ebd76093a.jpg)

**Comment by mooreniemi** [comment]
http://stackoverflow.com/questions/8737693/self-join-eager-loading-recursively

**Comment by mooreniemi** [comment]
still problematic:

```erb
$("<%= @parent_id ? "#comment-#{@parent_id} " : '#commentable' %>.reply-target").last().append(
  "<%= escape_javascript(render :partial => 'comments/form', :locals => { :comment => @new_comment }) %>"
).focus();

```

**Comment by mooreniemi** [comment]
merged into staging at ecd3207c66a970639f0b11122bbfd8433fa7222e
